### ISSUE TITLE
Support test running for library projects

### ISSUE THREAD

**Issue by lukaville** [MarathonLabs/marathon#111]
Currently `testApplicationOutput` param is required and `marathon-gradle-plugin` throws an exception if you try to apply it to a library project:

```
A problem occurred configuring project ':components:SomeUiPart'.
> com.android.build.gradle.internal.api.LibraryVariantOutputImpl_Decorated cannot be cast to com.android.build.gradle.api.ApkVariantOutput
```

**Comment by Malinskiy** [comment]
@lukaville How do you plan to test UI without the support from the framework? Unless I'm mistaken you can't test UI [without an application](https://developer.android.com/training/testing/espresso/basics)

**Comment by lukaville** [comment]
@Malinskiy You can run UI test for library project in the same way like for app project.

If you run `:some-library:assembleDebugAndroidTest` task it will assemble test apk with **library** itself and with **tests**: `some-library-debug-androidTest.apk`.

Then if you install the apk you can run the tests like this: `adb shell am instrument -w -r -e debug false -e class 'com.library.SomeTest' com.library.test/android.support.test.runner.AndroidJUnitRunner`. By default, I think it will use `android.app.Application` but it's customizable by using manifest or custom runner.

**Comment by rasskazovaleksey** [comment]
Espresso can't but it's possible with uiAutomator. Would be a nice feature

**Comment by lukaville** [comment]
Espresso works fine with library projects, so it's not a problem

**Comment by tagantroy** [comment]
Fixed #112
